src/test/test_dash.cpp

```diff
@@ -238,6 +238,4 @@ struct ExceptionInitializer {
     ~ExceptionInitializer()
     {
     }
-};
-
-BOOST_GLOBAL_FIXTURE( ExceptionInitializer );
+};
```

Comment:
revert this

Reply:
Hmm... looks like I was kind of wrong about this one, we should instead move the whole exception thing to `test_dash_main.cpp` now I think, see https://github.com/UdjinM6/dash/commit/806a4ff53ac1d56084511b921ec7d1a626740dd7.

Reply:
https://github.com/dashpay/dash/pull/2763/commits/9d48df7e1888a54ed543e09a1724eca7ee155cf4